pkg/registry/apps/statefulset/storage/storage.go

```diff
@@ -92,3 +115,88 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"sts"}
 }
+
+type ScaleREST struct {
+	registry statefulset.Registry
+}
+
+// ScaleREST implements Patcher
+var _ = rest.Patcher(&ScaleREST{})
+
+// New creates a new Scale object
+func (r *ScaleREST) New() runtime.Object {
+	return &extensions.Scale{}
```

Comment:
> Oh yeah, totally agree, but if you want this to work now, that's what you'd have to do. Otherwise, this actually working with other components will have to wait.

It's not a valid solution to "work" now.  That isn't working and doesn't fulfill the intent of the autoscaler.  If the infrastructure needs additional effort to be made to work, then that's necessary.

If we had caught the previous dummy in time, we would have stopped it before it got out the door.